checks.d/network.py

```diff
@@ -4,6 +4,7 @@
 # stdlib
 import re
 import subprocess
+from subprocess import check_output
```

Comment:
@maatthc This line seems to be causing tests to fail. Would you mind confirming when you have a moment?

ImportError: cannot import name check_output